infrastructure/deploy.sh

```diff
@@ -138,6 +138,21 @@ done
 # Copy ingress config to top level so it can be applied.
 cp deploy/ci_ingress.tf .
 
+# Check if a new ccdl-ubuntu ami will be needed for this region
+if [[ $(aws ec2 describe-images --region $TF_VAR_region --owners 589864003899 --filters 'Name=name,Values=ccdl-ubuntu-18.04-*' --query 'length(Images)') -eq 0 ]]; then
+    echo "No ccdl-ubuntu-18.04 AMI found for this region, creating a new one"
+
+    # Find most recent ccdl-ubuntu ami from us-east-1
+    template_ami_id=$(aws ec2 describe-images --region us-east-1 --owners 589864003899 --filters 'Name=name,Values=ccdl-ubuntu-18.04-*' --query 'sort_by(Images,&CreationDate)[-1].ImageId' --output text)
```

Comment:
These are some long lines. Makes it harder to read. When bash commands get really long like this I like using `\` to put one option per line. I believe there's some examples of this in this file.